nodewatcher/plugins/sge.py

```diff
@@ -71,3 +72,9 @@ def lockHost(hostname, unlock=False):
         run_sge_command(command)
     except subprocess.CalledProcessError:
         log.error("Error %s host %s", "unlocking" if unlock else "locking", hostname)
+
+
+def is_node_down():
+    """Check if node is down according to scheduler"""
+    # ToDo: to be implemented
+    return False
```

Comment:
I'd leave this with `return None` or pass so we don't accidentally call it

Reply:
actually returning False is done to disable this logic for sge and torque. nodewatcher will still call is_node_down but the node will never result as down (and this preserves the previous behaviour)